.gitignore

```diff
@@ -28,3 +28,4 @@ tsconfig.tsbuildinfo
 
 .pnpm-store/
 **/node_modules
+coverage
```

Comment:
I noticed an additional coverage directory. After switching to Vitest, will the test results display the UT coverage?

Reply:
Theoretically yes, but we don't have a way to run all the tests on a single node, so technically the coverage report will need some "merging" logic after coverage generation from each test runner.